readthedocs/builds/models.py

```diff
@@ -448,6 +478,7 @@ class Build(models.Model):
     exit_code = models.IntegerField(_('Exit code'), null=True, blank=True)
     commit = models.CharField(
         _('Commit'), max_length=255, null=True, blank=True)
+    config = JSONField(_('Configuration used in the build'), default={})
```

Comment:
Using a mutable object as default it's a problem. You need to change it to `dict` function.

Reference: https://docs.djangoproject.com/es/2.1/ref/models/fields/#django.db.models.Field.default

Reply:
I read this https://github.com/dmkoch/django-jsonfield/issues/125, but we can change it just to be 100% sure

Reply:
Yes. Change it, please. That's not the default behavior..